src/main/java/io/jenkins/plugins/appcenter/releasenotes/ReleaseNotes.java

```diff
@@ -0,0 +1,6 @@
+package io.jenkins.plugins.appcenter.releasenotes;
+
+import hudson.ExtensionPoint;
+import hudson.model.Describable;
+
+public abstract class ReleaseNotes implements ExtensionPoint, Describable<ReleaseNotes> { }
```

Comment:
What's the point of this class? Is it just to identify a type? Why not use an interface instead?